src/Components/Projects/Projects.tsx

```diff
@@ -1,6 +1,99 @@
 
+import { useContext, useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { AuthContext } from "../../Context/AuthContext";
+import axios from "axios";
+import NoData from "../../Shared/NoData/NoData";
+
+
 export default function Projects() {
+
+  const navigate = useNavigate();
+  const {baseUrl , requstHeaders} : any = useContext(AuthContext);
+  const [projectList , setProjectList] = useState([]);
+  const [isLoding , setIsLoding] =useState(false);
+
+
+    // *************** to get all projects *****************
+    const getCategoryList = ()=>{
+
+      setIsLoding(true)
+      axios.get(`${baseUrl}/Project/manager` , 
+      {
+        headers: requstHeaders ,
+      })
+      .then((response)=>{
+        console.log(response.data);
+        setProjectList(response?.data.data)
+        setIsLoding(false)
+      
+      }).catch((error)=>{
+        // error(error?.response?.data?.message || "Not Found Categorys")
+        console.log(error.response.data.message);
+        setIsLoding(false)
```

Comment:
Well in this Line and in Line 28 instead of setting the setLoding twice and repeat yourself, you can consider using finally like the following [
.finally(()=> {
        setIsLoding(false);
      })
]